MagneticField/GeomBuilder/test/python/testMagGeometry.py

```diff
@@ -9,33 +9,8 @@
 
 process.load("FWCore.MessageLogger.MessageLogger_cfi")
 
-process.MessageLogger = cms.Service("MessageLogger",
-    cerr = cms.untracked.PSet(
```

Comment:
Why do you have to change this test file? It is not run in production, and unless I'm missing something does not produce compiler warnings. These tests .pys are meant to be run only and explicitely to produce this output. 
Please revert the original file in the PR as this change will jeopardize running these tests for debugging in the future.